cfme/tests/integration/test_cfme_auth.py

```diff
@@ -352,3 +353,1134 @@ def _cleanup():
         for group in retrieved_groups:
             if group.exists:
                 group.delete()
+
+
+@pytest.mark.manual
+@test_requirements.auth
+@pytest.mark.tier(2)
+def test_ldap_password_plaintext():
+    """
+    Test that LDAP password is not logged in plaintext in evm.log.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1/3h
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.auth
+@pytest.mark.tier(1)
+def test_black_console_ipa_ntp():
+    """
+    Try to setup IPA on appliance when NTP daemon is stopped on server.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1/4h
+        caseposneg: negative
+        setup:
+            1. Have IPA server configured and running
+                - https://mojo.redhat.com/docs/DOC-1058778
+        testSteps:
+            1. ssh into IPA server stop NTP daemon
+            2. ssh to appliance and try to setup IPA
+                - appliance_console_cli --ipaserver <IPA_URL> --ipaprincipal <LOGIN>
+                    --ipapassword <PASS> --ipadomain <DOMAIN> --iparealm <REALM>
+        expectedResults:
+            1. NTP daemon stopped
+            2. Command should fail; setting up IPA unsuccessful
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@test_requirements.auth
+def test_black_console_ipa():
+    """
+    Test setting up IPA authentication with invalid host settings
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Configuration
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/6h
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.auth
+@pytest.mark.tier(3)
+def test_update_ldap_updates_login():
+    """
+    Change user/groups attribute in  ldap domain server.
+    E.g change user display name
+    Verify authentication fails for old display name
+    Verify authentication for new display name for the user.
+    Verify changing cache_credentials = True
+    entry_cache_timeout = 600
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Configuration
+        caseimportance: low
+        initialEstimate: 1/2h
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.auth
+@pytest.mark.tier(3)
+def test_saml_verify_user_login():
```

Comment:
Why isn't this case removed from test_manual.py in this PR? Looks like this is the test case for:
`def test_saml_verify_user_login_with_and_without_correct_groups_added_to_saml_server():`

Reply:
I must have missed this. I'll check for duplicates and remove what should have been removed.